.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/googleapis"]
+	path = third_party/googleapis
+	url = https://github.com/googleapis/googleapis.git
```

Comment:
Glad to see that importing the proto file this way works! In my previous attempts, I copied the file from googleapis locally in api/google/grpc/status.proto but had issues with the server code that was serializing the response at runtime.